kore/src/Kore/Parser/Parser.hs

```diff
@@ -84,41 +82,34 @@ BNF definition:
 
 The @meta-@ version always starts with @#@, while the @object-@ one does not.
 -}
-sortParser :: Parser Sort
-sortParser = do
+objectSortParser :: Parser Sort
+objectSortParser = do
     identifier <- idParser
     c <- ParserUtils.peekChar
     case c of
         Just '{' -> actualSortParser identifier
         _        -> return (SortVariableSort $ SortVariable identifier)
   where
     actualSortParser identifier = do
-        sorts <- inCurlyBracesListParser sortParser
+        sorts <- inCurlyBracesListParser objectSortParser
         return $ SortActualSort SortActual
             { sortActualName = identifier
             , sortActualSorts = sorts
             }
 
-{-|'validateMetaSort' checks that a @meta-sort@ is well-formed.
 
-Relevant BNF definitions:
+metaSortParser :: Parser Sort
+metaSortParser = do
+    sort <- keywordBasedParsers
```

Comment:
Could we extract this, so that `metaSortParser` is all written at one level of detail?
```.hs
metaSortKeywordParser :: Parser Sort
metaSortKeywordParser =
    keywordBasedParsers
        (map
            (show &&& return . metaSort)
            metaSortsListWithString
        )
```